common/scripted_triggers/wc_his_character_triggers.txt

```diff
@@ -129,12 +141,48 @@ is_kilrogg_trigger = {
 	exists = character:10005 # Kilrogg
 	this = character:10005 # Kilrogg
 }
+is_drekthar_trigger = {
+	exists = character:10100 # Drek'thar
+	this = character:10100 # Drek'thar
+}
+is_draka_trigger = {
+	exists = character:10020 # Draka
+	this = character:10020 # Draka
+}
+is_nazgrel_trigger = {
+	exists = character:10658 # Nazgrel
+	this = character:10658 # Nazgrel
+}
 
 # Dwarves
 is_magni_trigger = {
 	exists = character:17001 # Magni Bronzebeard
 	this = character:17001 # Magni Bronzebeard
 }
+is_muradin_trigger = {
+	exists = character:17002 # Muradin Bronzebeard
+	this = character:17002 # Muradin Bronzebeard
+}
+is_brann_trigger = {
+	exists = character:17028 # Brann Bronzebeard
+	this = character:17028 # Brann Bronzebeard
+}
+is_moira_trigger = {
+	exists = character:17029 # Moira Bronzebeard
+	this = character:17029 # Moira Bronzebeard
+}
+is_dagran_trigger = {
+	exists = character:12002 # Dagran Thaurissan
+	this = character:12002 # Dagran Thaurissan
+}
+is_kurdran_trigger = {
+	exists = character:22002 # Kurdran
+	this = character:22002 # Kurdran
+}
+is_falstad_trigger = {
```

Comment:
Again, I need to know the reference. I would make his beard bigger and balding.

Reply:
Falstad is always depicted as having a braided mustache and braided beard. No such beards exist within the game files, so I thought the triple braided beard would be a suitable approximation.

As with Kurdran, I think the hairstyle is less balding and more of an intentionally shaved down ponytail. I didn't want to give him the same hair as Kurdran, so I chose the next best thing.

A lot of these issues come down to a matter of the difference in art style between CK3 and Warcraft, it seems. I've more been prioritizing the spirit of the characters as opposed to their exact models as they appear in game to avoid issues like all dwarves looking exactly the same or making them seem bland and unimpressive compared to how they appear in official sources. If I had access to the massive ponytails and huge, multi-braided beards that WoW has, I would totally use them, but without them I feel like I need to take a little creative liberty with some of these characters.

Reply:
> Falstad is always depicted as having a braided mustache and braided beard. No such beards exist within the game files, so I thought the triple braided beard would be a suitable approximation.

For his look, I guess it's important to have a giant beard. Many remember him from HotS.